UPGRADE-3.x.md

```diff
@@ -14,6 +14,10 @@ version of aws/aws-sdk-php is automatically inferred from the installed package.
 The configuration nodes "sonata_media.cdn.cloudfront.region" and "sonata_media.cdn.cloudfront.version"
 are required when aws/aws-sdk-php 3.x is installed.
 
+### Support for NelmioApiDocBundle > 3.6 is added
+
+Since version 3.x, support for nelmio/api-doc-bundle > 3.6 is added. Controllers for NelmioApiDocBundle v2 were moved under `Sonata\MediaBundle\Controller\Api\Legacy\` namespace and controllers for NelmioApiDocBundle v3 were added as replacement. If you extend them, you must ensure they are using the corresponding inheritance.
```

Comment:
This document already have headers that indicate the version which these upgrade notes belong to.
```suggestion
Support for nelmio/api-doc-bundle > 3.6 is added. Controllers for NelmioApiDocBundle v2 were moved under `Sonata\MediaBundle\Controller\Api\Legacy\` namespace and controllers for NelmioApiDocBundle v3 were added as replacement. If you extend them, you must ensure they are using the corresponding inheritance.
```

Reply:
@phansys Do we need `Support for nelmio/api-doc-bundle > 3.6 is added.`? It is duplicated from header.

Reply:
I guess there is no need to duplicate this sentence.